backend/app/controllers/tasks.js

```diff
@@ -69,37 +69,12 @@ module.exports = {
                 .where(Project.id.equals(req.params.id))
             );
 
+            // Should this really throw an error if there are no tasks? It's not a big deal.
```

Comment:
yeah, probably right. Because if nothing was found then some weirdness would happen. But we should probably take out this comment if its not needed